nominal/ts.py

```diff
@@ -433,3 +433,11 @@ def from_flexible(cls, ts: str | datetime | IntegralNanosecondsUTC) -> Self:
         if isinstance(ts, str):
             ts = dateutil.parser.parse(ts)
         return cls.from_datetime(ts)
+
+
+_MIN_TIMESTAMP = _SecondsNanos(seconds=0, nanos=0)
+_MAX_TIMESTAMP = _SecondsNanos(seconds=9223372036, nanos=854775807)
```

Comment:
Maybe we should make `_SecondsNanos` more first class: make it public, named appropriately as e.g. `Timestamp` or similar, and put it in the function signatures.